src/index.js

```diff
@@ -4,6 +4,9 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+const title = <h1>Pokemon Pokedex Encyclopedia!</h1>;
```

Comment:
index.js should usually be kept clean mate, so you can move this h1 to the App.js file